src/LICENSE

```diff
@@ -1,14 +0,0 @@
-Video for Linux Two API Specification
```

Comment:
I'm not sure we can remove the license file...
the code does not derived from v4l2 api specification?

Reply:
@knorth55 I did not actually remember I removed this line, so we should keep it

Reply:
Done, resolved